plugins/updater/src/updater.rs

```diff
@@ -1363,6 +1371,21 @@ impl PathExt for PathBuf {
     }
 }
 
+#[cfg(windows)]
+fn escape_nsis_current_exe_arg(arg: &&OsStr) -> String {
+    let mut arg = arg.to_string_lossy().to_string();
+
+    /* if arg.contains('"') {
+        arg = arg.replace('"', r#""""#);
+    } */
+
+    if arg.contains(' ') || arg.contains('\t') || arg.contains('/') {
```

Comment:
Do we need to escape `/`? (by the way, just from testing, even if you add a surrounding quote at `/SOME_RANDOM_ARGS`, NSIS's `GetOptions` function seems to still treat that as the end of the previous option)

Reply:
Yes, I added this because without it `app.exe path/to --whatever` breaks on the /

Didn't think about args starting with a /

Reply:
Yeah me neither, no symbol worked so I went back to quotes. That definitely worked for me in the updater though, will double check when I'm back home, maybe I reverted too much.

Reply:
Hmm never mind, I think it works in `ShellExecuteW` 😂

![image](https://github.com/user-attachments/assets/237418d5-58fa-4377-b37b-d20a5dfade56)

When I execute it from the powershell, `DetailPrint "$CMDLINE"` gives me this which doesn't have the quotes in them

![image](https://github.com/user-attachments/assets/4db2ed39-50eb-42f1-bea3-56c1d27e21ca)